src/config/env.ts

```diff
@@ -44,7 +44,8 @@ const envVars: EnvVars = {
 	SENTRY_DSN: process.env.SENTRY_DSN,
 	JIRA_LINK_TRACKING_ID: process.env.JIRA_LINK_TRACKING_ID,
 	PROXY: getProxyFromEnvironment(),
-	GITHUB_REPO_URL: "https://github.com/atlassian/github-for-jira"
+	GITHUB_REPO_URL: "https://github.com/atlassian/github-for-jira",
+	GITHUB_DOMAIN: process.env.GHE_HOST || "github.com"
```

Comment:
How do we switch between GitHub and GHE? are we going to run a separate app?

Reply:
Don't know.  This is just what was already in the code but made it global.  I'm guessing at some point we'll have a value in the DB or something?

Reply:
Looks like it was some experimental code. For GHE it shouldn't be global parameter, because each GHE host can have its own domain name. However I don't mind leaving it as long as we don't use it. (cc @thombergs Do you know anything about these GHE params?)

Reply:
yeah, good point.  I can add a comment to it.  It could be that the idea is we have an app instance for each GHE customer?  I dunno...

Reply:
Shouldn't it be `GITHUB_HOSTNAME`?  https://superuser.com/a/59094

Reply:
@mboudreau ?

Reply:
@mboudreau that is something that we'll be decided upon during the investigations for GHE. We may find that GHE customers have one GitHub app that is installed on every org and/or we may have GHE customers that ask each org admin to create their own GH app. In the latter scenario, we could have multiple instances per customer. Still need to think through how we want to handle this.